variables.tf

```diff
@@ -197,6 +197,12 @@ variable "addons_depends_on" {
   default     = null
 }
 
+variable "bootstrap_self_managed_addons" {
```

Comment:
If this is of type `bool`, we postfix all `bool` variables with `_enabled`. Can you please update to follow that convention?